File: src-model/model/celestials/Star.java
package model.celestials;


import model.celestials.parts.Orbit;
import model.celestials.parts.Sphere;

public class Star extends CelestialBody {

	public Star(String name, Orbit orbit, Sphere sphere) {
		super(name, orbit, sphere, CelestialBodyType.STAR);
	}
}

File: src/game/model/celestials/Star.kt
package game.model.celestials

import game.model.celestials.parts.Orbit
import game.model.celestials.parts.Sphere

class Star(
        name: String,
        orbit: Orbit,
        sphere: Sphere
) : CelestialBody(name, orbit, sphere) {
    override fun glows() = true
}